samza-kafka/src/main/scala/org/apache/samza/config/KafkaConfig.scala

```diff
@@ -366,16 +366,14 @@ class KafkaConfig(config: Config) extends ScalaMapConfig(config) {
     val properties = new Properties()
 
     if (isStreamMode) {
-      properties.putAll(ImmutableMap.of(
-        "cleanup.policy", "compact",
-        "segment.bytes", String.valueOf(segmentBytes),
-        "max.message.bytes", String.valueOf(maxMessageBytes)))
+      properties.put("cleanup.policy", "compact")
```

Comment:
The putAll method has an ambiguous issue: https://github.com/scala/bug/issues/10418

So just moving to single line puts.